framework/src/st_things/things_stack/utils/things_rtos_util.c

```diff
@@ -22,83 +22,49 @@
 #include <sched.h>
 
 #include "things_rtos_util.h"
+#include "logging/things_logger.h"
 
 #define TAG "[things_rtos_util]"
 
-static char thingsStackthreadName[THINGS_STACK_MAX_INDEX][40] = { {"THINGS_STACK_PING"},	/*THINGS_STACK_PING_THREAD */
-	{"THINGS_STACK_RESETLOOP"},		/*THINGS_STACK_RESETLOOP_THREAD */
-	{"THINGS_STACK_CICONN_INIT"},	/*THINGS_STACK_CICONNETION_INIT_THREAD */
-	{"THINGS_STACK_CICONN_WAIT"},	/*THINGS_STACK_CICONNETION_WAIT_THREAD */
-	{"THINGS_STACK_WIFIPROV_SET"},	/*THINGS_STACK_WIFI_PROV_SET_THREAD */
-	{"THINGS_STACK_CLOUD_REFRESH"},	/*THINGS_STACK_CLOUD_REFRESH_THREAD */
-	{"THINGS_STACK_PRESENCE_NOTI"},	/*THINGS_STACK_PRESENCE_NOTI_THREAD */
-	{"THINGS_STACK_SERVEREXCE_LOOP"},	/*THINGS_STACK_SERVEREXCETUE_LOOP_THREAD */
-	{"THINGS_STACK_BASE_TIME_OUT"},	/*THINGS_STACK_BASE_TIME_OUT_THREAD */
-	{"THINGS_STACK_OICABORT"},		/*THINGS_STACK_OICABORT_THREAD */
-	{"THINGS_STACK_WIFI_JOIN"},		/*THINGS_STACK_WIFI_JOIN_THREAD */
-	{"THINGS_STACK_UPDATE_THREAD"},	/*THINGS_STACK_FOTA_THREAD */
-	{"THINGS_STACK_MAX_INDEX"}		/*THINGS_STACK_MAX_INDEX */
-};
+#define THINGS_STACK_PTHREAD_STACK_DEFAULT		2048
+#define THINGS_STACK_PTHREAD_DEFAULT_PRIORITY	100
+#define THINGS_STACK_PTHREAD_DEFAULT_POLICY		2
```

Comment:
Why don't you use SCHED_XX instead of 2? Noone knows what's the meaning of 2.

Reply:
OK. i will fix.